unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Analytics/BIWAnalytics.cs

```diff
@@ -104,10 +104,12 @@ public static void EndScenePublish(SceneMetricsModel sceneLimits, string success
         SendEditorEvent("end_scene_publish", events);
     }
 
-    public static void SceneLimitsOverPassed(SceneMetricsModel sceneLimits)
+    public static void SceneLimitsOverPassed(SceneMetricsModel sceneUsage, SceneMetricsModel sceneLimits)
```

Comment:
"overpass" is a single word. Beware of wording here, as "overpass" is negative and "passed" is positive. I see that this method also sends "limit_passed" and this can contradict the meaning depending on intent. 

Maybe we can find a less confusing set of words here?

```suggestion
    public static void SceneLimitsOverpassed(SceneMetricsModel sceneUsage, SceneMetricsModel sceneLimits)
```

Reply:
Changed to `SceneLimitsExceeded`